third_party/move/move-model/src/exp_rewriter.rs

```diff
@@ -52,6 +52,15 @@ impl<'env, 'rewriter> ExpRewriter<'env, 'rewriter> {
 }
 
 impl<'env, 'rewriter> ExpRewriterFunctions for ExpRewriter<'env, 'rewriter> {
+    fn rewrite_enter_scope<'a>(&mut self, vars: impl Iterator<Item = &'a (NodeId, Symbol)>) {
```

Comment:
Why have a NodeId in this API when its not used here?

Reply:
This is an implementation of a more general trait declared down below.